charts/ckan/templates/ckan/secret.yaml

```diff
@@ -0,0 +1,19 @@
+{{- $name := "ckan" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-%s-config" (include "common.names.fullname" $) $name | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ $name }}
+  namespace: {{ .Release.Namespace | quote }}
+stringData:
+  sysAdminUsername: {{ .Values.ckan.sysadmin.name | default "ckan_admin" | quote }}
+  sysAdminPassword: {{ .Values.ckan.sysadmin.password | default (randAlphaNum 42) | quote }}
```

Comment:
This will result in changing the password *every time*, see https://github.com/teutonet/teutonet-helm-charts/blob/main/charts/base-cluster/templates/monitoring/kube-prometheus-stack/grafana-secret.yaml#L15 on how to do it persistently